frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -18,6 +18,34 @@ describe("scenarios > question > new", () => {
     cy.signInAsAdmin();
   });
 
+  it.skip("data selector popover should not be too small (metabase#15591)", () => {
+    // Add 10 more databases
+    for (let i = 0; i < 10; i++) {
+      cy.request("POST", "/api/database", {
+        engine: "h2",
+        name: "Sample" + i,
+        details: {
+          db:
+            "zip:./target/uberjar/metabase.jar!/sample-dataset.db;USER=GUEST;PASSWORD=guest",
+        },
+        auto_run_queries: true,
+        is_full_sync: true,
```

Comment:
To avoid doing all the sync/fp, can you set it to false, and "this is a large db" - it would likely use less resources. Alternatively it could be stubbed, so we don't need the actual content of the databases